frontend/public/widget.js

```diff
@@ -5,10 +5,11 @@
     const loadWidget = () => {
         window.hiEventWidgetLoaded = true;
 
-        let scriptOrigin;
+       let scriptOrigin;
         try {
-            const scriptURL = document.currentScript.src;
-            scriptOrigin = new URL(scriptURL).origin;
+            const script = document.getElementById('hievent-widget-script');
```

Comment:
Something like this would be better as it makes it backwards compatible. As it stands old widget embeds would break.

```suggestion
let script = document.currentScript;
if (!script) {
   script = document.getElementById('hievents-widget-script');
}```